app/assets/javascripts/components/overview/overview_handler.jsx

```diff
@@ -50,10 +53,23 @@ const Overview = createReactClass({
     return ServerActions.fetch('tags', this.props.course_id);
   },
 
+  componentWillUnmount() {
+    if (this.timeout) {
+      clearInterval(this.timeout);
+    }
+  },
+
   storeDidChange() {
     return this.setState(getState());
   },
 
+  timeout: (function () {
+    if (this.state.course.level === 'Editathon') {
```

Comment:
@AmitJoki I don't this function is bound therefore the components state is not accessible by this function. Please look into the Travis Build log for more info.

Reply:
ah yes! Forgot only fat arrow functions make use of the contextual `this`. Have to bind now.